### PR TITLE
Second blog with additional feedback from AWS PM (Josh Bright)

### LINKED ISSUE #647: BYOK for B2B SaaS Operators
**Describe the blog post your would like to write**
B2B SaaS companies with Opensearch as backend data store can rapidly implement BYOK (Bring your own key)

**What is the title of the blog post?**
BYOK for B2B SaaS Operators

**Who are the authors?**
Pakshi Rajan

**What is the proposed posting date?**
ASAP

### PR DESCRIPTION
**Pull request by pakshi**
Signed-off-by: Pakshi Rajan <pakshi@titaniam.io>

### Description
This PR posts Titaniam's second blog - BYOK for B2B SaaS Operators https://github.com/opensearch-project/project-website/issues/647
 
### Issues Resolved
#647 

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by pakshi** [comment]
@stockholmux and @brijos 
This is the most recent version of the second blog. Please approve for publishing.

**Comment by krisfreedain** [comment]
> Hello, Is there something I should be doing to move this forward? If it is the case, please let me know. Best, Pakshi

Hello @pakshi - will be taking a look at this today.

**Comment by pakshi** [comment]
Thanks Kris

File: _posts/2022-02-18-partner-highlight-titaniam-byok.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "OpenSearch Partner Highlight: BYOK for B2B SaaS Operators using OpenSearch"
+authors:
+  - pakshirajan 
+date:   2022-02-18 01:01:01 -0700
```
Comment (krisfreedain) [inline _posts/2022-02-18-partner-highlight-titaniam-byok.md:6]: Hello @pakshi - can you update the date on this? While you're at it, please also update the filename to match the more recent date "2022-**02-18**-partner-highlight-titaniam-byok.md" 
thank you

**Review by krisfreedain — CHANGES_REQUESTED** [review]
a couple quick date edits. Thank you

**Comment by pakshi** [comment]
Creating a new one with date changes .. per @krisfreedain 's advice

File: _posts/2022-02-18-partner-highlight-titaniam-byok.md (hunk 1 above)
Comment (pakshi) [reply _posts/2022-02-18-partner-highlight-titaniam-byok.md]: @krisfreedain I made the changes

**Review by krisfreedain — CHANGES_REQUESTED** [review]
@pakshi - looks like one last edit with the image filename

File: _posts/2022-04-26-partner-highlight-titaniam-byok.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Steps to enabling data protection with BYOK for your OpenSearch based application.
+
+1. Install Titaniam Plugin for OpenSearch
+2. Choose which fields to protect. In a typical OpenSearch index, not all documents need to be secured and searched. You choose specific fields to secure.
+3. Test out your front end application works seamlessly.
+4. Create a key registry - specify which index should use which key and provide Titaniam with the credentials (in encrypted form) to access the corresponding keystores.
+5. Start loading data and querying it.
+
+![query-results](/assets/media/blog-images/2022-04-26-partner-highlight-titaniam/titaniam-opensearch-byok.png){: .img-fluid}
```
[the remark this answers is not in the document]
Comment (pakshi) [reply _posts/2022-04-26-partner-highlight-titaniam-byok.md]: Thanks @krisfreedain. Fixed it.

[the remark this answers is not in the document]
Comment (krisfreedain) [reply _posts/2022-04-26-partner-highlight-titaniam-byok.md]: thanks!

**Comment by pakshi** [comment]
Thanks everyone.

**Pull request merged**